### PR TITLE
Add --setecdefault documentation; fix typo

### PR DESCRIPTION
**Pull request by tst-eclamar**
Add "--setecdefault" documentation to webpages.  Fix small typo in geserveradmin --help.

Original issue: https://github.com/google/earthenterprise/issues/699

### CODE REVIEW

File: docs/geedocs/5.2.1/answer/6034724.html [hunk 1]
```diff
@@ -161,6 +161,9 @@ <h3>To publish a database using the command line: </h3>
 <li>Publish the database to GEE Server using <code>geserveradmin --publishdb db_name --targetpath target_path</code>. For example:
 <p><code>geserveradmin --publishdb /gevol/assets/Databases/SFMapDatabase.kmmdatabase/mapdb.kda/ver001/mapdb/ --targetpath http://myserver.org</code></p>
 </li>
+<li>Publish the database to GEE Server, specifying a default globe for GEEC, using <code>geserveradmin --publishdb db_name --targetpath target_path --setcedefault</code>. For example:
```
Comment (tst-nfarah) [inline docs/geedocs/5.2.1/answer/6034724.html]: may be 'specifying it as default globe...'

Comment (tst-rwildes) [reply docs/geedocs/5.2.1/answer/6034724.html]: 'specifying it as the default globe...'

File: docs/geedocs/5.2.1/answer/6034724.html [hunk 2]
```diff
@@ -161,8 +161,12 @@ <h3>To publish a database using the command line: </h3>
 <li>Publish the database to GEE Server using <code>geserveradmin --publishdb db_name --targetpath target_path</code>. For example:
 <p><code>geserveradmin --publishdb /gevol/assets/Databases/SFMapDatabase.kmmdatabase/mapdb.kda/ver001/mapdb/ --targetpath http://myserver.org</code></p>
 </li>
+<li>Publish the database to GEE Server, specifying it as the default globe for GEEC, using <code>geserveradmin --publishdb db_name --targetpath target_path --setcedefault</code>. For example:
```
Comment (tst-nfarah) [inline docs/geedocs/5.2.1/answer/6034724.html]: typo: setecdefault (instead of 'setcedefault')

File: docs/geedocs/5.2.1/answer/6034724.html [hunk 3]
```diff
@@ -161,8 +161,12 @@ <h3>To publish a database using the command line: </h3>
...
+<p><code>geserveradmin --publishdb /gevol/assets/Databases/SFMapDatabase.kmmdatabase/mapdb.kda/ver001/mapdb/ --targetpath http://myserver.org --setcedefault</code></p>
```
Comment (tst-nfarah) [inline docs/geedocs/5.2.1/answer/6034724.html]: typo: setecdefault (instead of 'setcedefault')

**Comment by tst-eclamar** [comment]
Need to process GP-1016 first.

**Pull request closed without merging**